packages/commons/src/main/util/ValidationUtil.ts

```diff
@@ -34,6 +34,10 @@ export function isValidEmail(email: string): boolean {
   return regExp.test(email);
 }
 
+export function isValidHandle(handle: string): boolean {
+  return /^@?[a-z_0-9.-]{2,256}$/.test(handle);
```

Comment:
This will validate handles like `bennyn` and `@bennyn`. I think we should stick to one.

Reply:
I copied this one over from webapp.

Reply:
Ah, we then cut the leading `@`. Okay.